test/interop/C/cmakelists/targetToLibToChpl/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 project(cmakeTest)
-cmake_minimum_required(VERSION 3.21)
+cmake_minimum_required(VERSION 3.18)
```

Comment:
Are these generated files from your new flag?

Reply:
No, these are the CMakeLists for the tests that then import the generated files. The generated files only create calls to `set`